lnwallet/interface_test.go

```diff
@@ -1227,6 +1227,13 @@ func testListTransactionDetails(miner *rpctest.Harness,
 			t.Fatalf("num confs incorrect, got %v expected %v",
 				txDetail.NumConfirmations, 0)
 		}
+
+		// We test that each txDetail has destination addresses. This ensures
+		// that even when we have 0 confirmation transactions, the destination
+		// addresses are returned.
```

Comment:
Can you check the address agains the burnOutput/burnTx?

Reply:
@halseth The burnOutput had no address, so I replaced it with a generated address from the miner harness node. 
```
minerAddr, err := miner.NewAddress()
if err != nil {
  t.Fatalf("unable to generate addrress: %v", err)
}
outputScript, err := txscript.PayToAddrScript(minerAddr)
if err != nil {
  t.Fatalf("unable to make output script: %v", err)
}
burnOutput := wire.NewTxOut(outputAmt, outputScript)
burnTX, err := alice.SendOutputs([]*wire.TxOut{burnOutput}, 2500)
```
There is an assertion for the generated miner address in the returned destination addresses. Coins being sent to that miner address doesn't seem to affect any other tests.

```
// We test that each txDetail has destination addresses. This ensures
// that even when we have 0 confirmation transactions, the destination
// addresses are returned.
if len(txDetail.DestAddresses) == 0 {
  t.Fatalf("0 conf transactions should display destination addresses")
}

// Assert that the returned destination addresses are the ones we
// expect.
var match bool
for _, addr := range txDetail.DestAddresses {
  if addr.String() == minerAddr.String() {
    match = true
    break
  }
}
if match != true {
  t.Fatalf("minerAddr: %v should have been a dest addr", minerAddr)
}
```

Reply:
Looks better! 👍